app/uk/gov/hmrc/mobilepaye/domain/taxcalc/P800Summary.scala

```diff
@@ -50,6 +50,13 @@ object P800Summary {
         case _                   => Option(p800Summary)
       }
 
+    def notOlderThanSixWeeks(p800Summary: P800Summary): Option[P800Summary] = {
+      p800Summary.datePaid match {
+        case None => None
+        case Some(date) => if (date.plusWeeks(6).plusDays(1).isAfter(LocalDate.now())) Option(p800Summary) else None
```

Comment:
You don't need the plusDays(1) as you are doing an isAfter. So it will be doing anything that is a millsecond after the date + 6 weeks.

Reply:
I thought the same initially, but used the tests to make sure.

Going by the example in the ticket: _"The time should be six weeks from today, e.g., today being Friday 4 October I would expect the card to be removed on Saturday 16 November"_
 
So if the date paid was Monday 2nd September, we shouldn't get a result today and if it was Tuesday 3rd then we should. I've manually entered those dates in the tests and it works as expected.